summoner-tui/src/Summoner/Tui.hs

```diff
@@ -175,6 +181,11 @@ appNew dirs = App
                 then loopWhileInactive ev newForm
                 else continue newForm
 
+    keyTriggersAutofill :: V.Key -> Bool
```

Comment:
Could we please add the documentation on which keys it triggers :slightly_smiling_face:

Reply:
Hi, thanks for this. "KBS" key wasn't obvious to me too. I've added a comment.
I'm assuming you meant comment by "documentation", unless there is another place that we keep such documentation.